.github/renovate.json5

```diff
@@ -12,6 +12,7 @@
     // manually bumping deps
     "@biomejs/biome",
     "@types/node",
+    "preact-render-to-string", // https://github.com/withastro/astro/pull/10200
```

Comment:
`preact-render-to-string` had many updates since, is it still buggy? Maybe it wasn't a bug and I didn't realize, but it definitely did seem like a regression.

Reply:
Yeah it was still buggy. Renovate updated to latest and it returns an array of strings. I'm not sure why though, maybe we're not calling `render` right if it's still not fixed.